src/createRoom.js

```diff
@@ -212,7 +212,18 @@ export async function _waitForMember(client, roomId, userId, opts = { timeout: 1
  * can encrypt to.
  */
 export async function canEncryptToAllUsers(client, userIds) {
-    const usersDeviceMap = await client.downloadKeys(userIds);
+    let usersDeviceMap;
+    try {
+        usersDeviceMap = await client.downloadKeys(userIds);
+    } catch (ex) {
+       if (ex.httpStatus === 404) {
+            console.log("Got 404 when fetching keys for users, not encrypting room");
+            // The endpoint to fetch keys doesn't exist: force unencrypted.
+            // See: https://github.com/vector-im/riot-web/issues/13598
+            return false;
+       }
+       throw ex;
```

Comment:
We might just want to assume the user has no possible devices and instead return `false` (homeserver lacks support).

Reply:
Yup, that was the original cut but wondering how much we care about things like matrix.org being flaky, triggering this exception, causing false positives.

Reply:
Great question for product :)